src/promptflow/promptflow/executor/_line_execution_process_pool.py

```diff
@@ -356,6 +356,8 @@ def run(self, batch_inputs):
                         for _ in range(self._n_process)
                     ],
                 )
+            except PromptflowException as e:
+                raise e
```

Comment:
No need to get the variable, we can simply do this:
```
except PromptflowException:
    raise
```

Reply:
got it, fixed